### PR TITLE
docs[patch]: Adds docs feedback text

### PR DESCRIPTION
**Pull request by jacoblee93**

### CODE REVIEW

File: docs/core_docs/src/theme/Feedback.js [hunk 1]
```diff
@@ -161,6 +164,33 @@ export default function Feedback() {
...
+      siteConfig.customFields.supabaseUrl,
+      siteConfig.customFields.supabasePublicKey
+    );
+    const { error } = await supabase.from("feedback_details").insert({
+      feedback_id: feedbackId,
+      details,
+    });
+    if (error) {
+      console.error("Failed to add feedback details", error);
+      throw error;
```
Comment (bracesproul) [inline docs/core_docs/src/theme/Feedback.js:189]: Do we want to actually throw this error in production? I think it would be better to just silently fail.

Comment (bracesproul) [reply docs/core_docs/src/theme/Feedback.js:189]: Adding on that, I'd wrap this function in a try/catch so if something else goes wrong it doesn't throw, and instead just silently logs the error and returns.

File: docs/core_docs/src/theme/Feedback.js [hunk 2]
```diff
@@ -161,6 +164,33 @@ export default function Feedback() {
     setFeedbackSent(true);
   };
 
+  const handleFeedbackDetails = async (e) => {
+    e.preventDefault();
+    if (!feedbackId) {
+      setFeedbackDetailsSent(true);
```
Comment (bracesproul) [inline docs/core_docs/src/theme/Feedback.js:170]: if `feedbackDetailsSent` is true do we want to return or allow the user to submit multiple times?

Also I don't understand why we're returning here & setting true if there is no ID?

File: docs/core_docs/src/theme/Feedback.js [hunk 3]
```diff
@@ -110,7 +111,9 @@ const getIpAddress = async () => {
 
 export default function Feedback() {
   const { setCookie, checkCookie } = useCookie();
+  const [feedbackId, setFeedbackId] = useState(null);
```
Comment (bracesproul) [inline docs/core_docs/src/theme/Feedback.js:114]: Might make more sense to store the ID in the cookie which is tracking feedback, this way they can't spam it, and their ID will persist between refreshes (for 24hrs)

**Review by bracesproul — COMMENTED** [review]
Couple questions/nits. otherwise lgtm

File: docs/core_docs/package.json [hunk 4]
```diff
@@ -45,6 +45,7 @@
     "process": "^0.11.10",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "uuid": "^10.0.0",
```
[the remark this answers is not in the document]
Comment (jacoblee93) [reply docs/core_docs/package.json:48]: Sure

File: docs/core_docs/src/theme/Feedback.js (hunk 1 above)
Comment (jacoblee93) [reply docs/core_docs/src/theme/Feedback.js:189]: It'll just show up in the console which I think is fine, probably don't need both though

File: docs/core_docs/src/theme/Feedback.js (hunk 2 above)
Comment (jacoblee93) [reply docs/core_docs/src/theme/Feedback.js:170]: If there's no id there, you're in a bad state and best thing to do is mark as done and no-op

File: docs/core_docs/src/theme/Feedback.js (hunk 3 above)
Comment (jacoblee93) [reply docs/core_docs/src/theme/Feedback.js:114]: They can't visually get back into that state anyway

**Pull request merged**